signingscript/src/signingscript/sign.py

```diff
@@ -966,6 +966,9 @@ def _xpi_signing_options(fmt):
         cose_algorithms = algos.split("_")
         if not cose_algorithms or set(cose_algorithms) - {"PS256", "ES256", "ES384", "ES512"}:
             raise SigningScriptError(f"Unsupported format {fmt}")
+    elif fmt == "webextension":
+        cose_algorithms = ["ES256"]
+        digest = "SHA1"
```

Comment:
This should presumably switch to sha2 eventually?

Reply:
I assume so, but this matches what AMO currently does, so willdurand and lgreco wanted to be consistent with that.

Reply:
Yeah, it'd be better to keep `sha1` for now.